src/plugins/kibana/server/lib/case_conversion.js

```diff
@@ -0,0 +1,15 @@
+const _ = require('lodash');
+
+module.exports = {
+  convertToSnakeCase: function (object) {
```

Comment:
Probably will be useful as soon as someone adds and API with a GET endpoint though. I don't see any harm in leaving it in, do you?